pkg/apis/kops/componentconfig.go

```diff
@@ -228,9 +228,15 @@ type KubeAPIServerConfig struct {
 	// The OpenID claim to use as the user name.
 	// Note that claims other than the default ('sub') is not guaranteed to be unique and immutable.
 	OIDCUsernameClaim *string `json:"oidcUsernameClaim,omitempty" flag:"oidc-username-claim"`
+	// Prefix prepended to username claims to prevent clashes with existing
+	// names (such as 'system:' users).
+	OIDCUsernamePrefix *string `json:"oidcUsernamePrefix,omitempty" flag:"oidc-username-prefix"`
```

Comment:
Great docs. You mind updating them to follow the go style guide? https://blog.golang.org/godoc-documenting-go-code

Sentence starts with the struct member name.

Reply:
We are trying to clean these up as you can see many are not correct.